<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HWTablemodelApp</class>
 <widget class="QWidget" name="HWTablemodelApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HWTablemodelApp</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="classEdit">
       <property name="placeholderText">
        <string>班级</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameEdit">
       <property name="placeholderText">
        <string>姓名</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="hwEdit">
       <property name="placeholderText">
        <string>作业名称</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="insertBt">
       <property name="text">
        <string>插入数据</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectBt">
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateBt">
       <property name="text">
        <string>更新</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteBt">
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="commitBt">
       <property name="text">
        <string>递交</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
